metacatalog_corr/models.py

```diff
@@ -75,6 +76,7 @@ class CorrelationMatrix(Base):
     identifier = sa.Column(sa.String(30), nullable=True)
     left_id = sa.Column(sa.Integer, nullable=False)
     right_id = sa.Column(sa.Integer, nullable=False)
+    warnings = sa.Column(sa.String(1000), nullable=True)
```

Comment:
I think we should keep the `CorrelationMatrix` table as small as possible. Remember, that we might well have millions of records here and each one will reserve a String field on 1000 characters disk space. From my point of view, the better approach is to use a many-to-many relationship here. That means, we only save a warning, if there is a warning and make these available through the Model. It would also be possible to reference the same warning by many records in the `CorrelationMatrix` table, which should make sense as similar datasets should produce similar Warnings. Then, we can also get rid of the 1000 characters limit, as some warnings might take more characters.

Reply:
You are right, I didn´t really think about the storage size of the table.